Automattic/jetpack
# cli: Fix build execa `stdio` null handling

## Proposed changes:

In stock node `child_process` and in default `execa`, passing null for a stream in `stdio` is equivalent to "pipe" for stdin/stdout/stderr.

But in the wrapper we define in the build command, we instead had it treating null as either "ignore" or "inherit" depending on the `-v` option. This non-standard interpretation crept into `helpers/list-project-files.js` and caused me some confusion when trying to debug something.

Let's bring the wrapper's behavior closer to the default. We still interpret "inherit" according to `-v`, but otherwise it should be the same now.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check the output from `jetpack build` with and without `-v`, it should look right.
